src/models/userRoleExtension.js

```diff
@@ -34,5 +34,9 @@ module.exports = {
 			type: String,
 			default: 'SYSTEM',
 		},
+		code: {
```

Comment:
@borkarsaish65 Add validation to make sure that if code is not provided during the creation of user extension we will throw a validation error

Reply:
done.